crates/types/src/traits/election.rs

```diff
@@ -38,7 +38,10 @@ pub trait Membership<TYPES: NodeType>:
     Clone + Debug + Eq + PartialEq + Send + Sync + Hash + 'static
 {
     /// generate a default election configuration
-    fn default_election_config(num_nodes: u64) -> TYPES::ElectionConfigType;
+    fn default_election_config(
+        num_nodes_with_stake: u64,
+        num_nodes_without_stake: u64,
```

Comment:
I'd prefer if we could not change the trait, but it's not a big deal if it's difficult to work around